Capstone/Translating /LanguageTableViewController.m

```diff
@@ -0,0 +1,62 @@
+//
+//  LanguageTableViewController.m
+//  Capstone
+//
+//  Created by Abel Kelbessa on 8/3/22.
+//
+
+#import "LanguageTableViewController.h"
+#import <Parse/Parse.h>
+
+@interface LanguageTableViewController ()
+
+@end
+
+@implementation LanguageTableViewController {
+    PFUser *currentUser;
+}
+
+- (void)viewDidLoad {
+    self->currentUser = [PFUser currentUser];
+}
+
+- (IBAction)didTapEnglishForNew:(id)sender {
+    self->currentUser[@"newLanguage"] = @"English";
+    [self->currentUser saveInBackground];
+}
+
+- (IBAction)didTapArabicForNew:(id)sender {
+    self->currentUser[@"newLanguage"] = @"Arabic";
+    [self->currentUser saveInBackground];
+}
+
+- (IBAction)didTapGermanForNew:(id)sender {
+    self->currentUser[@"newLanguage"] = @"German";
+    [self->currentUser saveInBackground];
+}
+
+- (IBAction)didTapSpanishForNew:(id)sender {
+    self->currentUser[@"newLanguage"] = @"Spanish";
+    [self->currentUser saveInBackground];
+}
+
+- (IBAction)didTapEnglishForCurrent:(id)sender {
```

Comment:
what's the difference between current and new for these functions?

Reply:
As mentioned earlier, I had create a new class that had less code because this class was not showing up on Xcode. Sorry for the confusion.